src/pages/SignIn.js

```diff
@@ -20,12 +19,14 @@ const styles = {
 }
 
 const SignIn = () => {
+  const brand = getEnabledBrand()
+
   return h(HeroWrapper, { showMenu: false, showDocLink: true }, [
     div({ style: { maxWidth: 600 } }, [
       div({ style: { fontSize: 36, color: colors.dark(0.6) } }, ['New User?']),
-      div({ style: { fontSize: 36, marginBottom: '2rem' } }, [`${getAppName({ capitalizeThe: true })} requires a Google Account.`]),
+      div({ style: { fontSize: 36, marginBottom: '2rem' } }, [`${brand.name} requires a Google Account.`]),
       div({ style: { fontSize: 16, lineHeight: 1.5, marginBottom: '2rem' } }, [
-        `${getAppName({ capitalizeThe: true })} uses your Google account. Once you have signed in and completed the user profile registration step, you can start using ${getAppName()}.`
+        `Once you have signed in and completed the user profile registration step, you can start using ${brand.signInName}.`
       ]),
       h(SignInButton),
       !isAnvil() && !isElwazi() && div({ style: styles.warningNoticeContainer }, [
```

Comment:
It seems like this warning text could also go into the brands structure.

Reply:
I thought about that myself but I don't have a great idea about how to structure the stuff in `SignIn.js` in a way to could enable a non-software engineer persona to modify it so decided on punting on that as well.